komodo-relational-commands/src/main/java/org/komodo/relational/commands/server/WkspStatusServerManager.java

```diff
@@ -55,6 +55,9 @@ public static WkspStatusServerManager getInstance(final WorkspaceStatus wsStatus
      * @throws KException if error occurs
      */
     public static void reset() throws KException {
+        if (instance == null)
```

Comment:
I don't expect the shell to be multiuser any time soon. It has been updated to ensure tests pass successfully etc... but I have done little thinking around how it would move forward in that area.